redaxo/src/addons/mediapool/pages/media.php

```diff
@@ -14,6 +14,16 @@
 
 $media_manager = rex_addon::get('media_manager')->isAvailable();
 
+if (!$media_manager) {
```

Comment:
Ich bin kein fan davon variablen ausserhalb des files zundefinen in denen sie genutzt werden.

Alternativ würd ich lieber diese 5 zeilen in beiden subpages haben